man/docker-stats.1.md

```diff
@@ -6,14 +6,19 @@ docker-stats - Display a live stream of one or more containers' resource usage s
 
 # SYNOPSIS
 **docker stats**
+[**-a**|**--all**[=*false*]]
 [**--help**]
-CONTAINER [CONTAINER...]
+[**--no-stream**[=*false*]]
+[CONTAINER...]
 
 # DESCRIPTION
 
 Display a live stream of one or more containers' resource usage statistics
 
 # OPTIONS
+**-a**, **--all**=*true*|*false*
```

Comment:
Hm, but we have them in the other parts (I know we had issues without blank lines in some docs(